test/test_files/tinysnb/order_by/multi_label.test

```diff
@@ -12,14 +12,14 @@
 325
 325
 
--NAME MultiLabelTest2
--QUERY MATCH (a:person)-[e:marries|:workAt|:mixed]->(b:person) return a.ID, b.ID order by ID(e) desc LIMIT 4
--ENUMERATE
----- 4
-7|8
-3|5
-0|2
-10|2
+#-NAME MultiLabelTest2
```

Comment:
I think we should add support to order by internal ID. Can u open an issue here.

Reply:
Opened issue #1212